File: src/cujae/inf/citi/om/data/DepotMDVRP.java
package cujae.inf.citi.om.data;

import java.util.ArrayList;

/* Clase que modela los datos de un depósito en el MDVRP*/

public class DepotMDVRP extends Depot{
	
	private ArrayList<Integer> listAssignedCustomers;

	public DepotMDVRP() {
		super();
		listAssignedCustomers = new ArrayList<Integer>();
	}
	 
	public DepotMDVRP(ArrayList<Integer> listAssignedCustomers) {
		super();
		this.listAssignedCustomers = listAssignedCustomers;
	}

	public ArrayList<Integer> getListAssignedCustomers() {
		return listAssignedCustomers;
	}

	public void setListAssignedCustomers(ArrayList<Integer> listAssignedCustomers) {
		this.listAssignedCustomers = listAssignedCustomers;
	}
}

File: _BHCVRP_Python_Version/cujae_inf_citi_om/_vrp_data/DepotMDVRP.py
from _vrp_data import Depot

# Clase que modela los datos de un depósito en el MDVRP

class DepotMDVRP(Depot):
    
    def __init__(self, list_assigned_customers=None):
        super().__init__()
        self._list_assigned_customers = list_assigned_customers if list_assigned_customers is not None else []

    def get_list_assigned_customers(self):
        return self._list_assigned_customers

    def set_list_assigned_customers(self, list_assigned_customers):
        self._list_assigned_customers = list_assigned_customers